packages/devtools_app/lib/src/config_specific/drag_and_drop/drag_and_drop.dart

```diff
@@ -112,9 +109,9 @@ class DragAndDrop extends StatefulWidget {
 class DragAndDropState extends State<DragAndDrop> {
   final _dragging = ValueNotifier<bool>(false);
 
-  NotificationsState notifications;
+  NotificationsState? notifications;
```

Comment:
can this be late instead?

Reply:
I would not. It is public field. So it can be accessed before didChangeDependencies, that set it.

Reply:
didChangeDependencies will be called directly after initState the first time the state is created, so it should be safe to assume `notifications` will be initialized anywhere it is accessed. If it is null, we are going to throw an error anyway when `activeState!.notifications!` is called, so making this late will just throw a late initialization error instead of a null operator error

Reply:
done